incubator/kolide-fleet/templates/service.yaml

```diff
@@ -0,0 +1,34 @@
+---
+kind: "Service"
+apiVersion: "v1"
+metadata:
+  annotations:
+{{- if .Values.pod.annotations }}
+{{ toYaml .Values.service.annotations | indent 4 }}
+{{ end }}
+    {{ if eq .Values.service.type "LoadBalancer" }}service.beta.kubernetes.io/external-traffic: "OnlyLocal"{{ end }}
```

Comment:
This if statement is unnecessary. The annnotation can be added with `service.annotations`. However, the annotation is obsolete with the field `externalTrafficPolicy` on the service spec. Add that and make it configurable.